aidb/config/config.py

```diff
@@ -150,6 +150,14 @@ def column_by_service(self) -> Dict[str, BoundInferenceService]:
     '''
     column_service = dict()
     for bound_service in self.inference_bindings:
+      actual_output_columns = bound_service.binding.output_columns
+      len_copied = len(bound_service.service.copied_input_columns)
```

Comment:
better to put 'copied_input_columns' inside 'bound_service.binding'?

Reply:
We cannot do so because we need to access it inside the inference service.

Reply:
Can you explain this a bit more @continue-revolution ?

Reply:
It is challenging to know which output row correspond to which input row.

One input row, for example, a frame to be sent to a detector, may produce 0, 1, or multiple output rows (in this example, multiple objects).

Copying input outside of inference service (inside bound inference service) might also be an option, if inference service return a list of pandas dataframes where each dataframe correspond to one input row (if no output produce, place en empty row).

Might be a better option, if you agree then I'll do it.

Reply:
What are the pros an cons of the two methods?